code/_globalvars/lists/objects.dm

```diff
@@ -47,4 +47,6 @@ GLOBAL_LIST_INIT(active_diseases, list()) 			//List of Active disease in all mob
 
 GLOBAL_LIST_EMPTY(mob_spawners) 		    // All mob_spawn objects
 
-GLOBAL_LIST_EMPTY(explosive_walls)
\ No newline at end of file
+GLOBAL_LIST_EMPTY(explosive_walls)
+
+GLOBAL_LIST_INIT(engine_beacon_list, list())
```

Comment:
Woudlnt `GLOBAL_LIST_EMPTY` be better here

Reply:
Yep. Done.